src/main/resources/api.yaml

```diff
@@ -313,6 +316,12 @@ paths:
           - Only need a valid and unexpired credential.
         - critical:
           - The credential must be valid and be issued in the last 5 minutes.
+      parameters:
+        - name: "cookie"
```

Comment:
It doesn't look correct, please read https://swagger.io/docs/specification/authentication/cookie-authentication/

Reply:
Thx and noticed(I tried this method before but not succeed at that time, so I switch to the "brute-force" header reading method). However, in order not to block others' process, I suggest to fix it in next PR.

Reply:
Fixed.